model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -27,10 +27,10 @@ data class AnalyzerConfiguration(
      * Enable the analysis of projects that use version ranges to declare their dependencies. If set to true,
      * dependencies of exactly the same project might change with another scan done at a later time if any of the
      * (transitive) dependencies are declared using version ranges and a new version of such a dependency was
```

Comment:
Would there be a similar problem with `skipExcluded` ?

Reply:
Isn't that already handled with the last commit?

Reply:
Right, I've overlooked that one.